### PR TITLE
Add license inspect command documentation and changelog

### PR DESCRIPTION
**Pull request by mkeeler**
Also reformatted another changelog entry to add a missing newline.

### CODE REVIEW

**Review by rboyer — APPROVED** [review]
LGTM

File: website/content/commands/license.mdx [hunk 1]
```diff
@@ -52,11 +56,67 @@ Usage: consul license <subcommand> [options] [args]
   For more examples, ask for subcommand help or view the documentation.
 
 Subcommands:
-    get      Get the current license
-    put      Puts a new license in the datacenter
-    reset    Resets the license in the datacenter
+    get        Get the active license
+    inspect    inspect a signed license
```
Comment (markan) [inline website/content/commands/license.mdx:60]: Maybe capitalize "Inspect" for consistency

**Review by markan — APPROVED** [review]
One minor typo, otherwise looks good.

File: website/content/commands/license.mdx (hunk 1 above)
Comment (mkeeler) [reply website/content/commands/license.mdx]: Yes it should. Thanks. Its updated now.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/381137.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit af3ffdf4c88de5a7d2df2ea687c4b2959cc8f15f onto `release/1.10.x` succeeded!

**Pull request merged**